openpdf/pom.xml

```diff
@@ -51,6 +51,11 @@
             <version>${jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
Why add another dependency? Perhaps test scope only?

Reply:
Yes, forgot it while making changes

Reply:
Fixed